src/domModule.js

```diff
@@ -0,0 +1,199 @@
+import {
+  ProjectModule,
+} from './projectModule';
+import {
+  todo,
+} from './TODO_factory';
+import {
+  TodoModule,
+} from './todoModule';
+import {
+  todoForm,
+} from './forms';
+
+const projects = ProjectModule.returnAllProjects();
+const formHolder = document.querySelector('#form-holder');
+let currentProject = 0;
```

Comment:
This error is also pretty straight forward to fix. The error has already told us how to fix it :)